docs/content/docs/test/Runner.md

```diff
@@ -672,16 +672,16 @@ instrumentation {
         local {
             tryToReRunOnTargetBranch = false
             targets {
-                api27 {
-                    deviceName = "API27"
+                api28 {
+                    deviceName = "API28"
                     scheduling {
                         quota {
                             retryCount = 1
                             minimumSuccessCount = 1
                         }
 
                         testsCountBasedReservation {
-                            device = new LocalEmulator("27", 27, "Android_SDK_built_for_x86")
+                            device = new LocalEmulator("28", 28, "AOSP_on_IA_Emulator")
```

Comment:
I've checked the image locally and see another state: `Model = Android_SDK_built_for_x86_64`